lib/project_types/extension/features/argo_serve.rb

```diff
@@ -22,6 +23,10 @@ def call
         end
       end
 
+      def resource_url
+        super || ExtensionProject.current(force_reload: true).resource_url
```

Comment:
✨ Excellent, nice use of SmartProperties' ability to override getters and only fallback to the value on disk or autogeneration if `--resourceUrl` wasn't passed.